src/lua/httpc.lua

```diff
@@ -585,6 +585,7 @@ curl_mt = {
 
             if not opts.chunked then
                 resp.url = url_with_params
+                resp.decoders = decoders
```

Comment:
Maybe it would be better to start the field from an underscore (so it'll be excluded from the autocompletion) and add `__serialize` that skips fields that start from the underscore. Or capture `decoders` in a locally created function and pass it to `decode_body` in a separate argument.

Reply:
Now the field is named `_decoders` and it is excluded from the serialization.